store/storetest/thread_store.go

```diff
@@ -349,24 +349,6 @@ func testThreadStorePopulation(t *testing.T, ss store.Store) {
 		require.Equal(t, int64(0), th.UnreadReplies)
 	})
 
-	t.Run("Empty participantID should not appear in thread response", func(t *testing.T) {
-		newPosts := makeSomePosts()
-		opts := store.ThreadMembershipOpts{
-			Following:             true,
-			IncrementMentions:     false,
-			UpdateFollowing:       true,
-			UpdateViewedTimestamp: false,
-			UpdateParticipants:    true,
-		}
-		m, err := ss.Thread().MaintainMembership("", newPosts[0].Id, opts)
```

Comment:
With the the addition of the SeenReplyCount column, the `UserID` argument to MaintainMembership can't be empty (PostId being empty doesn't make sense either). This was never actually supported, and it seems to be check we missed previously. I've added a check to the MaintainMembership function and removed this test.